### ISSUE TITLE
Add-Reaction-Button and Action "3 dots"-Button besides the message do not fit with new version

### ISSUE THREAD

**Issue by stefanandres** [0x0049/Rocket.Chat.Dark#2]
Appreciate your new version.

I've installed it via the manual custom css mehtod (because I'm deploying rc via docker compose and this is easier for me then) and I noticed 2 things:

1. The topic is now white instead of dark
2. the add-reaction button/icon is now out of place, as well as the 3-dot action menu (see attachment in the middle at the right)

The black color also makes it easy to overlook

<img width="1426" alt="screen shot 2018-02-18 at 09 17 26" src="https://user-images.githubusercontent.com/2674469/36349816-d89ef6ac-148c-11e8-81ce-4a1abb8db632.png">

**Comment by eryg-kai** [comment]
Thanks for the report! I pushed up a fix for cozy & compact modes.

I can't seem to replicate the black message action icons. What version of Rocket.Chat are you running?

Is there a setting to get that version of the topic/channel header? Mine doesn't have any icons and instead I have a sidebar on the right. I looked around for a setting but I haven't been able to find anything yet.

**Comment by stefanandres** [comment]
I'll try to create a reproducable example this week

**Comment by Otiel** [comment]
I'm having the same issue for the channel header. Using the Windows client Rocket.Chat+ 2.10.5 on Rocket.Chat 0.61.0.

**Comment by eryg-kai** [comment]
Thanks @Otiel I'll check out the Windows client when I can, I only tested on the web version which is probably why I haven't been able to duplicate the issues.

**Comment by stefanandres** [comment]
Here's how to reproduce the white topic and black action icons, if you have docker-compose:
```
cd /tmp/
cat > docker-compose.yaml <<EOF
version: '2'

services:
  rocketchat:
    image: rocketchat/rocket.chat:latest
    restart: unless-stopped
    volumes:
      - ./uploads:/app/uploads
    environment:
      - MONGO_URL=mongodb://mongo:27017/rocketchat
      - MONGO_OPLOG_URL=mongodb://mongo:27017/local
      #- MAIL_URL=smtp://smtp.email
      - ROOT_URL=https://localhost
      - Accounts_UseDNSDomainCheck="false"
    depends_on:
      - mongo
    ports:
      - 3000:3000

  mongo:
    image: mongo:3.2
    restart: unless-stopped
    volumes:
     - ./data/db:/data/db
    command: mongod --smallfiles --oplogSize 128 --replSet rs0

  # this container's job is just run the command to initialize the replica set.
  # it will run the command and remove himself (it will not stay running)
  mongo-init-replica:
    image: mongo:3.2
    command: 'mongo mongo/rocketchat --eval "rs.initiate({ _id: ''rs0'', members: [ { _id: 0, host: ''localhost:27017'' } ]})"'
    depends_on:
      - mongo
EOF

docker-compose up -d
```

- then you have the latest rc running on http://localhost:3000
- register an admin account
- Put https://raw.githubusercontent.com/0x0049/Rocket.Chat.Dark/master/dist/dark.css into custom css
- Result:
<img width="1673" alt="screen shot 2018-02-20 at 18 31 29" src="https://user-images.githubusercontent.com/2674469/36439265-a482d690-166c-11e8-8c97-a380d9811250.png">

**Comment by eryg-kai** [comment]
Thanks for that! I think everything should be fixed now, but I haven't done super thorough testing yet, so I'll be going through it again later.

Also, I noticed a whole bunch of things were different so I verified my version from within RC this time rather than just checking what version my package manager said was installed and long story short my installing script was downloading 0.59.3 rather than 0.61.1 as I thought it was. :blush:
